src/utils/bibDetailUtils.ts

```diff
@@ -0,0 +1,7 @@
+export const displayRtl = (value: string) => {
```

Comment:
I think these functions could benefit from comments and more explicit naming. I also don't mind them moving to the bibUtils file to cut down on the number of utils files.